enterprise-suite/README.md

```diff
@@ -89,17 +83,19 @@ Install [yq](https://github.com/mikefarah/yq) if you don't have it yet:
     # or
     brew install yq                  
 
+Make sure the image versions in `enterprise-suite/values.yaml` are as desired.
+
 Then run the release script on a clean master checkout:
 
-    scripts/make-release.sh enterprise-suite
+    ./scripts/make-release.sh enterprise-suite
```

Comment:
Why is `./` needed?

Reply:
Not really needed.  I just put it there cuz a lot of people don't have '.' in their path.  I think I tried to cut'n'paste that line and it failed for me.

Leaving for now.  We can change later if need be.  (Cuz I think I made that tweak in other places too.)

Reply:
You don't need `.` in your PATH. `scripts/make-release.sh` should be valid in any `sh` based shell.